2fa-api/handlers/login.js

```diff
@@ -0,0 +1,45 @@
+const db = require("../db");
+const twilio = require("../twilio");
+
+const twilioClient = twilio.twilioClient;
+
+const login = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+
+    db.connection.query(
+      `SELECT id, phone from Users WHERE email='${email}' and password='${password}'`,
+      (error, results, _) => {
+        if (error) {
+          console.log(error);
+          return res.status(500).json({ error });
+        }
+
+        if (!results.length) {
+          return res
+            .status(200)
```

Comment:
why is this a `200`? what error code should it actually be? you're matching the error message rather than using the status code - thats incorrect

Reply:
fixed - made it a 401 unauthorized